am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/AccessManagementServiceIntegrationTest.java

```diff
@@ -21,7 +23,11 @@ public void testSetup() {
 
     @Test
     public void createQuery_whenCreatingResourceAccess_ResourceAccessAppearsInDatabase() {
-        ams.createResourceAccess(resourceId, "dsa");
+        ams.createResourceAccess(
+                resourceId,
+                "dsa",
+                new ExplicitPermissions(Permissions.CREATE, Permissions.READ, Permissions.UPDATE)
```

Comment:
plz extract them to constant

Reply:
Which ones? "Permissions.CREATE, Permissions.READ, Permissions.UPDATE" - this sequence?

Reply:
whole ExplicitPermissions object. its used 3 times

Reply:
what if an implementation of *ExplicitPermissions* change? you will have to change 3 places instead of one. It also improves readability as in current implementation it implies that its different for each test cases which is not true.

Reply:
done